packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -173,8 +176,8 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
 
             const miSdkApiTag = document.createElement('script');
             miSdkApiTag.setAttribute('type', 'text/javascript');
-            miSdkApiTag.setAttribute('src', 'https://app.mapsindoors.com/mapsindoors/js/sdk/4.32.0/mapsindoors-4.32.0.js.gz');
-            miSdkApiTag.setAttribute('integrity', 'sha384-pobsiCLAduarZklerr0Y1Yj0gkSVov1spL2XwAMLQ1lm3t6USIXIBAAMeJqiWXSc');
+            miSdkApiTag.setAttribute('src', 'https://app.mapsindoors.com/mapsindoors/js/sdk/4.34.0/mapsindoors-4.34.0.js.gz');
+            miSdkApiTag.setAttribute('integrity', 'sha384-PmlLSyZqDHBMT7mdLvzISBGszeCx5Bi7Emk5kb1Cmy2MA/SmgcjZvNsVAvqjw3ri');
```

Comment:
Merge main into this feature branch, since the SDK has been updated and there are some merge conflicts.